### PR TITLE
Renamed

### PR DESCRIPTION
**Pull request by StarTrooper08**
What is been changed ?
- "Action Bar" word is changed to "App Bar" as this issue is mentioned in issue no ankidroid#28 of ankidroiddocs.
- Naming convention in Material UI is also changed from Action Bar to App Bar.

### CODE REVIEW

File: manual.asc [hunk 1]
```diff
@@ -7,10 +7,10 @@
 
 == Introduction
 
-Thank you for using AnkiDroid, the Android client for the popular 
-http://ankisrs.net/[Anki] spaced repetition system.
```
Comment (david-allison-1) [inline manual.asc:11]: There's a lot of additional spacing changes in this diff.

Could you remove these and only include relevant content changes?

File: manual.asc [hunk 2]
```diff
@@ -56,22 +56,22 @@ Create deck :: Create a new empty deck
  . Choose a name for the deck, for example "New Japanese"
  . Add cards to it following the "Add" instructions above
 
-=== Action Bar
-At the top of each screen in AnkiDroid is the Action Bar, with buttons for performing various actions. 
-The following actions are available from the action bar in the deck list:
+=== App Bar
```
Comment (david-allison-1) [inline manual.asc:59]: Changing heading names may break links. I'm unsure if AsciiDoctor has functionality to preserve existing links, but we should look into it.

File: manual.asc [hunk 3]
```diff
@@ -163,8 +163,8 @@ From the deck list, if you tap the counts area you will be taken to the deck ove
 
 On this screen you can view a summary of the deck, build custom study sessions, rebuild / empty filtered decks, and change deck options. When visible, pressing the study button will take you to the study screen for that deck.
 
-=== Action bar
-The icons that are shown in the action bar depend on whether your deck is an ordinary deck or a filtered deck.
+=== App Bar
```
Comment (david-allison-1) [inline manual.asc:166]: Flagging heading change

File: manual.asc [hunk 4]
```diff
@@ -206,19 +206,19 @@ When you've looked at a card's question and remembered the answer, or decided
...
 
-To make reviewing faster, you can configure gestures (for example taps and swipes) to answer cards without using the buttons. 
+To make reviewing faster, you can configure gestures (for example taps and swipes) to answer cards without using the buttons.
 See the <<settings, preferences section>> for more information on configuring gestures.
 
-=== Action Bar
-The Action Bar at the top of the study screen has several buttons for performing various common actions. The number of buttons which are shown 
-is determined automatically by Android based on the size and resolution of your screen. If there is not enough space to show the button 
-for a given action, then the action will be available from the menu instead. If you are unsure what a button does, you can long-tap on it to 
+=== App Bar
```
Comment (david-allison-1) [inline manual.asc:214]: Flagging Heading change

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
Thanks! Change looks good.

A few nitpicks, to clean up the diff

Main issue would be working out if AsciiDoctor has any functionality to preserve heading names to ensure we don't break links. We may need to bite the bullet on this one, but it'd be ideal if there's functionality to fix it. I seem to recall something, but may be mistaken.

Could you rename the commit message into something a little more helpful for future users: `Rename 'action bar' to 'App Bar'`

File: manual.asc (hunk 1 above)
Comment (StarTrooper08) [reply manual.asc:11]: ok I will change the commit message

File: manual.asc (hunk 2 above)
Comment (mikehardy) [reply manual.asc:59]: I just read the manual, I was unable to find a way to manually specify the anchor that a section title should convert to, I think we just change it and move on

**Comment by mikehardy** [comment]
Superceded by #88

**Pull request closed without merging**